packages/databricks-vscode/src/bundle/BundleInitWizard.ts

```diff
@@ -34,9 +38,21 @@ export async function promptToOpenSubProjects(
         title: "Select the project you want to open",
     };
     const item = await window.showQuickPick<OpenProjectItem>(items, options);
-    if (!item) {
+    if (!item?.uri) {
         return;
     }
+
+    if (authProvider?.authType === "profile") {
+        const rootOverrideFilePath =
+            OverrideableConfigModel.getRootOverrideFile(item.uri);
+        await mkdir(path.dirname(rootOverrideFilePath.fsPath), {
+            recursive: true,
+        });
+        await writeFile(
+            rootOverrideFilePath.fsPath,
+            JSON.stringify({authProfile: authProvider.toJSON().profile})
+        );
+    }
```

Comment:
Maybe move this logic to the OverrideableModel?

Although I'm not even sure we need a concept of the "root" override file (at least not yet). In the "normal" scenario with only one profile per host we won't really need this override, right? After opening that folder we will log-in automatically. Having all this logic around root override seems like a lot of effort for not much benefit, unless we already plan to use it for a bunch of other things.

Reply:
We don't have a way to enforce 1 profile per host (and should not because we can have multiple auth methods). A lot of bricksfooders ended up creating multiple profiles for the same host when initializing multiple projects. The extension would then be in a login state until they manually reselected their already created profiles. 

This allows us to establish some communication method between the pre bundle init extension instance and post bundle init extension instance. Right now it is just this, but in future with env management, I can imagine having to communicate that information, or cluster information as well.